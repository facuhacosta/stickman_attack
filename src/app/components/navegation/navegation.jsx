import React from 'react';
import style from './navegation.module.scss';
  
export function Navegation() {

  return (
    <div className={style.navegation}></div>
  );
}