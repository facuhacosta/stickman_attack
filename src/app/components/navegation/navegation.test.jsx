import React from 'react';
import { render } from '@testing-library/react';
import { Navegation } from './navegation';

describe('Navegation Component', () => {

  // Matching Snapshots
  test('it should match the snapshot', () => {
    const { asFragment } = render(<Navegation />);
    expect(asFragment()).toMatchSnapshot();
  });

  // Unit tests

});