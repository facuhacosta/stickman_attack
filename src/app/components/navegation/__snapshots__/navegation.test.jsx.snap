// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navegation Component it should match the snapshot 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
